src/main/java/com/nnaltakyan/core/communications/domain/sentemail/task/SendEmailTask.java

```diff
@@ -0,0 +1,53 @@
+package com.nnaltakyan.core.communications.domain.sentemail.task;
+
+import com.nnaltakyan.core.communications.domain.email.EmailService;
+import com.nnaltakyan.core.communications.domain.sentemail.service.SentEmailService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.springframework.data.domain.Page;
+
+@Component
+@Slf4j
+@ConditionalOnProperty(prefix = "internal.tasks.send-email", name = "enabled")
+public class SendEmailTask {
+
+    private final SentEmailService sentEmailService;
+    private final EmailService emailService;
+    private final int pageSize;
+
+    public SendEmailTask(final SentEmailService sentEmailService, EmailService emailService,
```

Comment:
please add final keyword also for EmailService

Reply:
Done.